docs/articles/intro/index.rst

```diff
@@ -0,0 +1,66 @@
+Introduction
+************
+
+This is the Software Factory Blueprint.
+
+How to use this blueprint
+=========================
+The blueprint is not intended to be particularly useful on its own, rather it is a collection of
+generic descriptions, processes, how-tos et cetera.
+
+The typical user of this blueprint is someone who wants to run a project where a Software Factory is
+part of the project. The blueprint can then be used as a base and then be extended with
+project-specific info, or one can just pick and choose the relevant parts from the blueprint.
+
+.. only:: blueprint
```

Comment:
I don't like this. Could we write the content in general as  "the blueprint" and not "this blueprint" and assume that the blueprint is something that co-exists with a Deployment at most times. The few that reads the Blueprint in isolation should have gotten the description of the structure as part of the Blueprint anyway and will know that there is a Deployment missing.

The section that describes the structure to the reader, including nomenclature etc, doesn't need to be part of this change, but this change could assume there will be that kind of info later.